packages/panzer/dof-mgr/test/fe_assembly/test_fe_assembly_HEX.cpp

```diff
@@ -0,0 +1,62 @@
+// @HEADER
+// ************************************************************************
+//
+//                           Intrepid2 Package
+//                 Copyright (2007) Sandia Corporation
+//
+// Under terms of Contract DE-AC04-94AL85000, there is a non-exclusive
+// license for use of this work by or on behalf of the U.S. Government.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+// 1. Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright
+// notice, this list of conditions and the following disclaimer in the
+// documentation and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the Corporation nor the names of the
+// contributors may be used to endorse or promote products derived from
+// this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY SANDIA CORPORATION "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL SANDIA CORPORATION OR THE
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+// LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+// NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Questions? Contact Kyungjoo Kim  (kyukim@sandia.gov), or
+//                    Mauro Perego  (mperego@sandia.gov)
+//
+// ************************************************************************
+// @HEADER
+
+/** \file test_01.cpp
+    \brief  Test for checking orientation tools for hexahedral elements.
+    \author Created by Mauro Perego
+*/
+
+#include "test_fe_assembly_HEX.hpp"
+
+#include "Kokkos_Core.hpp"
+
+
+int main(int argc, char *argv[]) {
+
+  Kokkos::ScopeGuard kokkosScope (argc, argv);
+  Teuchos::GlobalMPISession mpiScope (&argc, &argv);
+  
+  const int r_val = Intrepid2::Test::feAssemblyHex<double,PHX::Device>(argc, argv);
```

Comment:
Why is this in the intrepid2 namespace? The code and test is in panzer? Could we change this? If this is code written by the user, we usually don't use the panzer namespace so users understand it is not part of the library. I would use something like panzer_test for the namespace.

Reply:
good catch! What about Discretization::Example::feAssemblyHex? The idea eventually is to move this in a "Discretization" package.

Reply:
I like that!